server/lib/search.js

```diff
@@ -106,17 +107,41 @@ const getSearchTermSQLConditions = (term, collectiveTable) => {
       sqlConditions = `AND ${collectiveTable ? `${collectiveTable}.` : ''}"slug" ILIKE :sanitizedTerm || '%' `;
     } else {
       sanitizedTerm = splitTerm.length === 1 ? sanitizeSearchTermForTSQuery(trimmedTerm) : trimmedTerm;
+      sanitizedTermNoWhitespaces = sanitizedTerm.replace(/ /g, '');
```

Comment:
:ice_cream: Better to remove all kinds of whitespace characters

```suggestion
      sanitizedTermNoWhitespaces = sanitizedTerm.replace(/\s/g, '');
```